### DISCUSSION TITLE
Development

### DISCUSSION THREAD

**Discussion by WEBDIENSTE** [deeztek/Hermes-Secure-Email-Gateway#80] [General]
Is the Hermes Gatewa, still in development?
If yes, when will Ubuntu 22.0 and Ubuntu 24.ü4 versions be released?

**Comment by deeztek** [comment]
Yes Hermes SEG is still in active development.  The plan is to introduce support for at least 22.04 and possibly 24.04 on the next release.

**Comment by edwardleehl7097** [comment]
Do you plan for clustering so that all settings, whitelisted, blacklisted list and etc can be shared each node?

**Comment by deeztek** [comment]
Clustering is on the horizon, however the current focus is mail server and groupware integration. Thanks!

**Reply by edwardleehl7097** [reply]
which means Hermes SEG can only run in single server only?
